packages/css/src/identifier.ts

```diff
@@ -16,7 +16,7 @@ function getDevPrefix(debugId: string | undefined) {
     parts.unshift(file && file !== 'index' ? file : dir);
   }
 
-  return parts.join('_');
+  return parts.join('_').replace(/([^\w\d])/gm, '_');
```

Comment:
This fix just replaces all non word/decimal characters with `_`, even `-` characters which are commonly used. Instead of just replacing characters we've previously opted to escape them using [cssesc](https://www.npmjs.com/package/cssesc). 

Not sure why that's not working here but needs to be investigated.